### PR TITLE
Entry for Finals [Electronics_Creed]

### PR DESCRIPTION
**Pull request by g-rohith**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

[] Registration Pull request.

[x] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/finalist/Electronics_Creed/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+type: finalist                   
+title: Team Electronics_Creed
+name: Start-up Booster
```
Comment (Uyadav207) [inline content/finalist/Electronics_Creed/index.md]: Name will be same as Title within " "

File: content/finalist/Electronics_Creed/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+description: We are young developers who accept difficult challenges.
+Team-Quote: Ideas for future technology
+Application_Type: Website
+---
```
Comment (Uyadav207) [inline content/finalist/Electronics_Creed/index.md]: Quotes "  ".   do not followed instructions

File: content/finalist/Electronics_Creed/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+We as a team focuses on boosting the confidence of young entrepreneur to pursue their dream. Unemployment has been a big problem in India because the competition is high and resources are less. Our project "Start-up Booster" lay emphasis on shortening the gap between a dream idea with its practical reality. Establishing a start-up at an ideal location with the ideal budget would ease this problem to some extent. So we are building a website that aids those who are planning to establish a start-up but don’t know how to really get going. Our web application will ask the start-up interested user to input various necessary data including some personal information like Name, Gender, Contact info, followed by their idea and the name of state and industry in which they want to prosper their start-up.
```
Comment (Uyadav207) [inline content/finalist/Electronics_Creed/index.md]: intro too long

File: content/finalist/Electronics_Creed/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+---
+
+# Project's Name
```
Comment (Uyadav207) [inline content/finalist/Electronics_Creed/index.md]: mention project name

File: content/finalist/Electronics_Creed/index.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+_**GitHub-Link**_ : https://github.com/rohith-4296/Code_For_Hackathon  
+
+_**Hosted-Link**_ : https://drive.google.com/drive/u/0/folders/1wT21Y7YattYg8LmPbvLwv0J98rgKSZhM <br>
+                    Video description - https://drive.google.com/file/d/1P_jb1Qw6jX28X5CTY5xIv3BlGHmaxKaS/view?usp=sharing
+
+
+---
+
+
+# Leader's Name :  Vaibhav Singh
```
Comment (Uyadav207) [inline content/finalist/Electronics_Creed/index.md]: name is place of Leader's name

File: content/finalist/Electronics_Creed/index.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+_**Role-In-Team**_  : Built web pages using his skills in JavaScript, HTML, CSS.
+
+_**Working-Stack**_ : HTML, CSS, Java Script
+
+_**University/School**_ : Bangalore Institute of Technology, Bangalore
+
+
+# Member1 Name :  Karan Pandey
```
Comment (Uyadav207) [inline content/finalist/Electronics_Creed/index.md]: Name in place of Member1 Name

**Review by Uyadav207 — CHANGES_REQUESTED** [review]
@g-rohith , Do changes asap.. Also give editable option to the Maintainer... else Even if i wish i can't  do the correction.

**Comment by g-rohith** [comment]
I have done the changes as mentioned.
Do I close this pull request and resend the PR

**Pull request merged**